projects/kit/components/progress/progress-label/progress-label.style.less

```diff
@@ -0,0 +1,24 @@
+:host {
+    position: relative;
+    display: inline-block;
+    line-height: 0;
+}
+
+::ng-deep progress {
+    width: 100%;
```

Comment:
Maybe we can use something like inline-flex to stretch it without ng-deep?

Reply:
done